cryptoapis/src/endpoint/bc_info.ts

```diff
@@ -1,8 +1,8 @@
 import { Requester, Validator } from '@chainlink/external-adapter'
-import { AdapterRequest, Config } from '@chainlink/types'
+import { ExecuteWithConfig, Config } from '@chainlink/types'
 import { DEFAULT_ENDPOINT } from '../config'
 
-export const Name = 'bc_info'
+export const Names = ['difficulty', 'height']
```

Comment:
Definitely important for backwards compatibility, but I wonder if in time these names should be housed in `ea-legos` since they are the specific data point to fetch.

In `ea-legos` it would map 'bc_info' => ['difficulty', 'height'].

So our Chainlink request would be for the data point `block_height` from `cryptoapis` which would then be transformed to call this API wrapper that knows how to make that HTTP request aka
```
{
  "endpoint": "bc_info",
  "path": "payload.headers"
}
```

I think this builds toward a future where data providers could just upload their whole API via an OpenAPI spec.

Don't think this is a right now thing, just thinking out loud.